lib/cdo/hamburger.rb

```diff
@@ -285,7 +285,7 @@ def self.get_header_contents(options)
 
     any_student_links = [
       {title: I18n.t("#{loc_prefix}my_dashboard"), url: CDO.studio_url("/home"), id: "header-student-home"},
-      {title: I18n.t("#{loc_prefix}course_catalog"), url: CDO.studio_url("/courses"), id: "header-student-courses"},
```

Comment:
Same with this one. I think this needs to stay

Reply:
Got it, thanks. FYI, this will still show as "Curriculum Catalog" on their text -- do you want it to show as "Course Catalog"?